### PR TITLE
File Requests

### PR DESCRIPTION
**Pull request by rossmeissl**
Hi @Jesus — Here's a start on adding support for File Requests, beginning with the `create` endpoint.

I'm not exactly sure how the error subtypes work, so I left that mostly empty.

### CODE REVIEW

**Comment by Jesus** [comment]
Hi, thanks for your contribution.

This looks good to merge but I don't have the time to go through it right now. Hopefully I'll find an evening next week to take care of this.

File: lib/dropbox_api/endpoints/file_requests/create_file_request.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    Path        = "/2/file_requests/create".freeze
+    ResultType  = DropboxApi::Metadata::FileRequest
+    ErrorType   = DropboxApi::Errors::CreateFileRequestError
+
+    # Create a file request for a given path.
+    #
+    # @param title [String] The title of the file request. Must not be empty.
+    # @param destination [String] The path of the folder in the Dropbox where uploaded files will be sent. For apps with the app folder permission, this will be relative to the app folder.
+    add_endpoint :create_file_request do |title, destination|
+      perform_request({
```
Comment (apurvis) [inline lib/dropbox_api/endpoints/file_requests/create_file_request.rb]: https://github.com/bbatsov/ruby-style-guide#no-braces-opts-hash

File: lib/dropbox_api/endpoints/file_requests/create_file_request.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        title: title,
+        destination: destination,
```
Comment (apurvis) [inline lib/dropbox_api/endpoints/file_requests/create_file_request.rb]: trailing comma

**Comment by Jesus** [comment]
Hi, I haven't forgotten about this I just haven't got the time yet.

I'll review & hopefully merge as soon as I get a chance.

**Comment by Jesus** [comment]
Thanks for your contribution and your patience.

**Comment by rossmeissl** [comment]
Thanks @Jesus !

**Pull request merged**